app/services/atmosphere/destroy_appliance_set.rb

```diff
@@ -0,0 +1,19 @@
+module Atmosphere
+  class DestroyApplianceSet
+    def initialize(appliance_set)
+      @appliance_set = appliance_set
+    end
+
+    def execute
+      @appliance_set.appliances.each do |appliance|
+        unless DestroyAppliance.new(appliance).execute
+          @appliance_set.errors.
+            add(I18n.t('appliance_seta.cannot_remote_appliance',
```

Comment:
Two typos ('seta' vs 'sets'; 'remote' vs. 'remove' - the second one requires a corresponding change in `locales/en.yml').